### ISSUE TITLE
Feature request : Encryption in DB

### ISSUE THREAD

**Issue by BirdInFire** [faldez/tanoshi#302]
I would like to ask a facultative encryption of the user data file (because since it's self hosted tell to the user we accept that we can"t read their data is good (some store 18+ manga)).

Thanks !

note : to not generate a problem if the FR https://github.com/faldez/tanoshi/issues/301 is accepted it would be nice to have the password  (for decryption in the browser) after the connection.

**Comment by faldez** [comment]
Do you mean encryption in the rows or the whole db?

**Comment by BirdInFire** [comment]
> Do you mean encryption in the rows or the whole db?

I was thinking something like all the file (but with administrator activation (like nexcloud server side encryption)).
Like that for those of us who have VPS who don't like 18+ content.
the vps (even if he do automatic scanning) they will be encrypted (but tanoshi can read it (of course otherwise it cannot do update every hour)).

I had asked to enter our own password but you can use a generated key stored somewere.

**Comment by faldez** [comment]
Ah you mean encrypt the comic book files?

**Comment by BirdInFire** [comment]
> Ah you mean encrypt the comic book files?

yeah and the user "followed list" and since it's server side encrypted technically if the attacker can read the db it can dencrypt it, but it's more like i said to avoid any "automated" scan from the VPS who d'ont like some hentai (loli & so on).
